### ISSUE TITLE
Calico v3.27.2 is not working with TigeraOperator v1.32.5

### ISSUE THREAD

**Issue by AymanZahran** [tigera/operator#3223]
## Expected Behavior
calico-node on ARM Instances are crashing with error
```
calico-node calico-node: error while loading shared libraries: libpcap.so.0.8: cannot open shared object file: No such file or directory
calico-node Calico node failed to start
```
AMD is working fine

## Steps to Reproduce (for bugs)
Deploy Tigera Operator v3.27.2 on EKS Arm NodeGroup

**Comment by Rajesh-Gurjar** [comment]
Hi @AymanZahran This looks similar to [https://github.com/projectcalico/calico/issues/8407](https://github.com/projectcalico/calico/issues/8407) the workaround is to create a symlink for libpcap.so.8 and the fix will be released in 3.27.3 soon.

**Comment by tmjd** [comment]
It sounds like this was due to issues with the Calico v3.27.3 images, I'm going to close this as it isn't something to address in the operator.
